.travis.yml

```diff
@@ -1,35 +0,0 @@
-language: java
-sudo: required
-dist: trusty
-#group: edge
-
-script:
-  - ./gradlew check
-cache:
-  directories:
-  - $HOME/.gradle
-
-before_install:
-  - export GRADLE_OPTS=-Xmx1024m
-
-matrix:
-  include:
-# JDK 7 no longer supported by Gradle 6+
-#    - jdk: openjdk7
-    - jdk: oraclejdk8 # JDK 1.8.0_131-b11
-    - jdk: oraclejdk9
-
-# Don't let Travis CI execute './gradlew assemble' by default
-# From https://github.com/reactive-streams/reactive-streams-jvm/pull/383
-install:
-# Display Gradle, JVM and other versions
-  - ./gradlew -version
-
-env:
-  global:
-  - TERM=dumb
-  - DEFAULT_TIMEOUT_MILLIS=300
```

Comment:
@sullis What about these TCK ENV vars?

Reply:
Not in use anymore?

https://github.com/reactive-streams/reactive-streams-jvm/search?q=DEFAULT_TIMEOUT_MILLIS